src/buyingcatalogue/values.yaml

```diff
@@ -362,6 +362,18 @@ ordapi:
       baseUriConfig:
         name: "bc-buyingcatalogue"
         key: issuer-url
+    email:
```

Comment:
do any of the .purchasing settings need adding as well, e.g. senderName/address?

Reply:
The have defaults at the moment. We haven't actually been given the content for the email address yet so the values at the moment are just placeholders. I can add these if you like? The ISAPI ones are commented out, but don't usually like leaving comments in as can cause clutter.